ghost-to-markdown/src/index.js

```diff
@@ -81,9 +83,29 @@ function convert(doc, useFigure) {
   }
 }
 
+function createPostHeaderTemplate(post) {
+
+  const { title, published_at, slug, feature_image, authors, tags } = post;
+
+  const headerTemplate = `---
+title: "${title}"
+date: "${published_at}"
+slug: "${slug}"
+feature_image: "${feature_image ?? ""}"
+author:
+  - name: ${authors[0].name}
+    slug: ${authors[0].slug}
```

Comment:
Is slug and the username the same thing? What do we need to map it to the Hashnode equivalent?

Reply:
Only the author slug is needed. Kris suggested added them in just to be on the safer side.

Reply:
Yeah, I figured it would be best to save this info if possible. I can see a lot of instances where authors will have different slugs / usernames on both platforms, like `quincylarson` on Ghost and `quincy` on HN.